mizar/dp/mizar/operators/endpoints/endpoints_operator.py

```diff
@@ -159,11 +159,16 @@ def update_scaled_endpoint_backend(self, name, spec):
         if ep is None:
             return None
         backends = set()
+        backend_ports = set()
```

Comment:
There is a problem with this. The backend_ports and backends may not have the same one-to-one mapping with sets. It can be ordered. It is better to associate a port with a backend in a separate class or simply a tuple. It's unlikely to happen, but it can happen.

Reply:
Thanks for the catch! I read up some more on Kubernetes services and it looks like we will need to do more work to support multi-port services.
https://kubernetes.io/docs/concepts/services-networking/service/#multi-port-services
In this case, I think we will instead need to have a one-to-one mapping between the frontend port and backend port.

This PR actually breaks the current e2e tests since we run multiple servers on different ports (TCP, HTTP, UDP tests) in a single container.